target/linux/generic/files/drivers/net/phy/rtl8366_smi.c

```diff
@@ -14,7 +14,7 @@
 #include <linux/delay.h>
 #include <linux/gpio.h>
 #include <linux/spinlock.h>
-#include <linux/skbuff.h>
+#include <linux/skbuff.h>RF
```

Comment:
RF ?

Reply:
i have no idea... looks like strange line ending???

Reply:
I think these are mistakenly pressed keys, they need to be removed.